capstone.html

```diff
@@ -0,0 +1,1062 @@
+					<ul>
+						<li>The PCB must adhere to the provided manufacturer constraints.</li>
+						<li>The PCB must be less than 100x100 mm in size.</li>
+					</ul>
+				</li>
+				<li class="list-group-item">
+					The rover must be able to move forward, backward, left, and right in response to commands
+					sent by the remote.
+				</li>
+			</ul>
+			<p class="mb-5">
+				Lastly, the rover must be stable and reliable. There should be no spontaneous movement, no wireless 
+				connection issues, and absolutely no sparks.
+			</p>
+
+			<h2 class="mb-3 fw-bold" id="pj-heading-procedure">Procedure</h2>
+			<h4 class="mb-3 fw-light">PCB Design</h4>
+
+			<ol class="mb-4 list-group list-group-numbered">
+				<li class="list-group-item">
+					Open KiCad 7.0 and create a new project.
+					<ul>
+						<li>
+							Name the project file using the following format: 
+							“<b>capstone_firstname_lastname.kicad_pro</b>”.
+						</li>
+					</ul>
+				</li>
+				<li class="list-group-item">
+					Create <b>Schematic A</b> in the project’s schematic editor.
+					<ul>
+						<li>You must use the symbols and corresponding footprints listed below:</li>
+						<ul>
+							<li>
+								Symbol: <b>Arduino_Nano_v3.x</b> <br>
+								Footprint Library: <b>Module</b> <br>
+								Footprint: <b>Arduino_Nano</b>
+							</li>
+							<li>
+								Symbol: <b>Conn_01x05_Female or Conn_01x05_Socket</b> <br>
+								Footprint Library: <b>Connector_PinSocket_2.54mm</b> <br>
+								Footprint: <b>PinSocket_1x05_P2.54mm_Vertical</b> <br>
+							</li>
+							<li>
+								Symbol: <b>NRF24L01_Breakout</b> <br>
+								Footprint Library: <b>RF_Module</b> <br>
+								Footprint: <b>nRF24L01_Breakout</b> <br>
+							</li>
+						</ul>
+						<li>
+							Note that we do not use a symbol for the joystick. Instead, we use a 1x5 female header. 
+							You must observe the physical pinout of the joystick and make sure it matches the pin 
+							wiring of the header in your KiCad schematic.
+						</li>
+					</ul>
+				</li>
+				<li class="list-group-item">
+					Before proceeding to the next step, make sure that you have done the following:
+					<ul>
+						<li>
+							Run the <b>Annotate Schematic</b> tool. <img class="img-fluid ms-1"
+							src=".\assets\images\project_8\ops_project_8_kicad_button_reference_designators.png">
+						</li>
+						<li>
+							Run the <b>Electrical Rules Checker (ERC)</b> tool. <img class="img-fluid ms-1"
+							src=".\assets\images\project_8\ops_project_8_kicad_button_electronic_rules_checker.png">
+						</li>
+						<ul>
+							<li>
+								There should be no violations except for <b>Error: Input Power pin not driven by 
+								any Output Power pins.</b>
+							</li>
+						</ul>
+						<li>
+							Make sure every symbol is assigned to the correct footprint.
+						</li>
+					</ul>
+				</li>
+				<li class="list-group-item">
+					Download this <a class="link-primary"
```

Comment:
Given the conventions that we have used to create new pages (basically, anything we host on google drive should actually be hosted locally), I believe that we would want to keep this file locally to the website.  I would probably actually create a new folder under `assets` that is relevant to this file, and put the file in here.

Reply:
Fixed

Reply:
I still see google drive links

Reply:
they should be modified to be similar to the edit you made to the quadruple half h driver pdf file